app/assets/stylesheets/buttons.scss

```diff
@@ -0,0 +1,21 @@
+// gradient buttons
+$gradient: linear-gradient(45deg, black, blue);
+$gradient2: linear-gradient(45deg, black, blue);
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.